cadquery/cq.py

```diff
@@ -2198,7 +2198,10 @@ def radiusArc(
         # Calculate the sagitta from the radius
         length = endPoint.sub(startPoint).Length / 2.0
         try:
-            sag = abs(radius) - math.sqrt(radius ** 2 - length ** 2)
+            sag = abs(radius)
+            r_2_l_2 = radius ** 2 - length ** 2
```

Comment:
Maybe add a comment explaining why this is needed

Reply:
Comment added